src/components/scenegraph/Toolbar.js

```diff
@@ -148,9 +181,59 @@ export default class Toolbar extends React.Component {
     let objects = this.state.objects;
     let objectChanges = [];
     let changedObjectsString = "";
-    for(var id in AFRAME.INSPECTOR.history.updates)
-      objectChanges.push([parseInt(id.replace("-obj", "")), AFRAME.INSPECTOR.history.updates[id]]);
+
+    for(var id in AFRAME.INSPECTOR.history.updates){
+      if (id.endsWith("-obj")){
+        objectChanges.push([parseInt(id.replace("-obj", "")), AFRAME.INSPECTOR.history.updates[id]]);
+      } else {
+        const objName = id.split("@")[0];
+        if (!('gltf-model' in AFRAME.INSPECTOR.history.updates[id])){
+          alert("Error: Please provide gltf-model for object with name: " + id);
```

Comment:
Should we exit out of this function here? Because the gltf-model isn't provided so we can't do anything further with this?